osu.Game.Tests/Visual/Online/TestSceneNewsOverlay.cs

```diff
@@ -2,22 +2,28 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Game.Overlays;
+using osu.Game.Overlays.News;
 
 namespace osu.Game.Tests.Visual.Online
 {
     public class TestSceneNewsOverlay : OsuTestScene
     {
-        private NewsOverlay news;
+        private TestNewsOverlay news;
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            Add(news = new NewsOverlay());
+            Add(news = new TestNewsOverlay());
             AddStep(@"Show", news.Show);
             AddStep(@"Hide", news.Hide);
 
             AddStep(@"Show front page", () => news.ShowFrontPage());
             AddStep(@"Custom article", () => news.Current.Value = "Test Article 101");
         }
+
+        private class TestNewsOverlay : NewsOverlay
+        {
+            public new void LoadAndShowChild(NewsContent content) => base.LoadAndShowChild(content);
```

Comment:
no usage of this?

Reply:
This is being used in upcoming PR for the news overlay for testing purposes. Guess it is better to keep that for when it is being used?

Reply:
generally we avoid adding any new functions without having test coverage.